EnglishFilter/sections/whitelist.txt

```diff
@@ -51,6 +51,9 @@ youtube.com#@##header.ytd-c4-tabbed-header-renderer:style(position: absolute !im
 ! https://github.com/AdguardTeam/PopupBlocker/issues/224
 ! https://github.com/AdguardTeam/AdguardFilters/issues/67111
 @@||ilfattoquotidiano.it^$extension
+! https://github.com/AdguardTeam/AdguardFilters/issues/79900
+@@||googletagservices.com/tag/js/gpt.js$domain=downdetector.it

```

Comment:
why removed redirect tule? It can be used here.
`||googletagservices.com/tag/js/gpt.js$script,redirect=googletagservices-gpt,domain=downdetector.it`